jmbitcoin/jmbitcoin/secp256k1_transaction.py

```diff
@@ -203,7 +205,7 @@ def sign(tx, i, priv, hashcode=SIGHASH_ALL, amount=None, native=False):
     or: (None, errormsg) in case of failure
     """
     # script verification flags
-    flags = set()
+    flags = set([SCRIPT_VERIFY_STRICTENC])
```

Comment:
Just a minor style comment: `s = set([value])` is a python2-ism (that periodically slips through in my code, too). The `s = {value}` form is better, visually, I think.

Reply:
But the code is totally OK and functional, of course, I don't mean that any change is necessary.